res/values-b+es+r419/strings.xml

```diff
@@ -27,45 +27,42 @@
     <string name="Yes">"Sí"</string>
     <string name="No">"No"</string>
     <string name="WAIT">"ESPERA…"</string>
-    <string name="Must_Specify_Name_">"Debes especificar un nombre "</string>
+    <string name="Must_Specify_Name_">"Debes especificar un nombre"</string>
 
-    <string name="Content_rating_of_the_app_may_change_when_finding_groups_">"El contenido de esta aplicación puede variar cuando se buscan grupos "</string>
+    <string name="Content_rating_of_the_app_may_change_when_finding_groups_">"El contenido de esta aplicación puede variar cuando se busquen grupos"</string>
 
-    <string name="_Rank_">" Posición "</string>
-    <string name="_Alias_">" Apodo "</string>
-    <string name="_Score_">" Puntuación "</string>
-    <string name="Score__">"Puntuación: "</string>
-    <string name="Recombine_">"Juntarse: "</string>
-    <string name="WINNER_">"GANADOR: "</string>
+    <string name="_Rank_">"Posición"</string>
+    <string name="_Alias_">"Apodo"</string>
+    <string name="_Score_">"Puntuación"</string>
+    <string name="Score__">"Puntuación"</string>
+    <string name="Recombine_">"Juntarse"</string>
+    <string name="WINNER_">"GANADOR"</string>
     <string name="IDLE">"INACTIVO"</string>
-    <string name="Name_Invalid_">"Nombre Inválido "</string>
-    <string name="Name_Taken_">"Nombre en Uso "</string>
+    <string name="Name_Invalid_">"Nombre inválido"</string>
+    <string name="Name_Taken_">"Nombre en uso"</string>
     <string name="CONNECTED">"CONECTADO"</string>
     <string name="JOINING___">"UNIÉNDOSE…"</string>
     <string name="PLAYING">"JUGANDO"</string>
     <string name="MULTIPLAYER">"MULTIJUGADOR"</string>
 
     <string name="JOIN">"UNIRSE"</string>
 
-
     <string name="Lobby_Name">"Nombre de la sala"</string>
-    <string name="Not_found_">"No encontrado "</string>
+    <string name="Not_found_">"No encontrado"</string>
 
     <string name="Hidden">"Oculto"</string>
 
-
     <string name="SERVER">"SERVIDOR"</string>
-    <string name="Joining_game_in_">"Entrando al juego "</string>
-    <string name="Game_Full">"Juego lleno"</string>
-    <string name="US_East">"Centroamérica"</string>
-    <string name="US_West">"Norteamérica"</string>
+    <string name="Joining_game_in_">"Entrando al juego"</string>
+    <string name="Game_Full">"La partida está llena"</string>
+    <string name="US_East">"Este de EE. UU."</string>
+    <string name="US_West">"Oeste de EE. UU."</string>
```

Comment:
You should base off first on the `res/values` (English file). Please return them as they were, because despite original values say "US East" and "US West" according to the English file these texts are already correct, both server names recently changed their names into English, and both translations "Centroamérica" (Central America) and "Norteamérica" (North America) were already correct.

```suggestion
      <string name="US_East">"Centroamérica"</string>
    <string name="US_West">"Norteamérica"</string>